shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -214,6 +214,12 @@ - (void)setTextInputState:(NSDictionary*)state {
   }
 }
 
+- (NSRange)validateSelection:(NSRange)range forText:(NSString*)text {
```

Comment:
nit: What about `clampSelection` (TBH I'm just hoping it gets some of those linewraps to squeak just under the column limit) ;)

Reply:
sure